### PR TITLE
Eth flow/updated contract to 0.0.0 rc4

### PR DESCRIPTION
**Pull request by alfetopito**
# Summary

Update ethflow contract to the split version of barn/prod

Placed a swap:
- tx https://goerli.etherscan.io/tx/0x7a8c475c84301f9ee977c70984d2136c500453c8ec4f1b707484b6949aaa8d0e
- order https://protocol-explorer.dev.gnosisdev.com/goerli/orders/0x1e76baa3eef0727665ab17cc8149b3c5312c26ec4ef1365340b3e3b032a593dcf55e40061fe3dffb7ae166b46a008dfc2ada510fffffffff

  # To Test

1. Place an order from this PR
* Should use the contract `0xF55E40061fe3dFfB7aE166B46A008dfc2aDA510f`

### CODE REVIEW

**Comment by alfetopito** [comment]
I need this change to unblock placing ethflow orders now that the backend has been updated.

Please post merge review

**Comment by elena-zh** [comment]
Hey @alfetopito , I have been able to place orders, contract address is correct. 
Just a tiny question: is this OK to see tons of red console logs when an onchain transaction is running? 
image

As far as I can understand, these logs are displayed until the TX is executed and the order is created on Explorer. So can we hide them at least?

File: package.json [hunk 1]
```diff
@@ -208,7 +208,7 @@
   "license": "GPL-3.0-or-later",
   "dependencies": {
     "@amplitude/analytics-browser": "^1.1.4",
-    "@cowprotocol/ethflowcontract": "cowprotocol/ethflowcontract.git#main-artifacts",
+    "@cowprotocol/ethflowcontract": "cowprotocol/ethflowcontract.git#0.0.0-rc4-artifacts",
```
Comment (anxolin) [inline package.json:211]: why aren't we making NPM packages and versioning them?

**Review by anxolin — COMMENTED** [review]
post-merge APPROVED

**Comment by alfetopito** [comment]
> Hey @alfetopito , I have been able to place orders, contract address is correct. Just a tiny question: is this OK to see tons of red console logs when an onchain transaction is running? image
> 
> As far as I can understand, these logs are displayed until the TX is executed and the order is created on Explorer. So can we hide them at least?

404s could be treated as non-errors, true
The other about insufficient liquidity, not sure it's related to ETH flow or something else.

**Comment by elena-zh** [comment]
> 404s could be treated as non-errors, true The other about insufficient liquidity, not sure it's related to ETH flow or something else.

Hey @alfetopito , here is the video with all errors I see after an order placement
https://www.loom.com/share/d15f6d076fba48c8938f79d9f2141e49

Console is totally in red, But I didn't change anything on the form after I had specified a token pair and an amount to trade.

File: package.json (hunk 1 above)
Comment (fedgiac) [reply package.json:211]: We (almost) don't use anything that is JS related, so we wanted to experiment with different release techniques.
If needed we can create a JS package, but it feels somewhat wrong to use NPM to just pin some non-js files to a specific version.

**Pull request merged**